lib/constants.go

```diff
@@ -1218,6 +1229,11 @@ var DeSoTestnetParams = DeSoParams{
 	// Jailed validators can be unjailed after a minimum of N elapsed epochs.
 	ValidatorJailEpochDuration: uint64(3),
 
+	// When registering, a validator must include a VotingSignatureBlockHeight within CurrentBlockHeight
+	// to CurrentBlockHeight + ValidatorVotingSignatureBlockHeightWindow blocks. This is to prevent
+	// validator registration replay attacks.
+	ValidatorVotingSignatureBlockHeightWindow: uint64(60),
```

Comment:
not sure if this value is exactly what we want, but I'm ok w/ it for now.

Reply:
Yeah, I don't know exactly what value should be here, and eventually this will move to a GlobalParamsEntry, but I initially just estimated 60.

Reply:
I updated it to 300.